test/project_deno/README.md

```diff
@@ -0,0 +1,32 @@
+# Project
+
+A test Gleam project.
+
+It covers these features:
+
+- Downloading packages
+  - Specified in config.dependencies
+- Importing packages
+  - Specified in config.dependencies
+- Compilation of Gleam code
+  - in src
+  - in test
+- Importing Gleam src code into test
+
+These features are not tested yet
+
+- Downloading packages
+  - Specified in config.dev-dependencies
+- Importing packages
+  - Specified in config.dev-dependencies
+- Compilation of locally defined JavaScript modules
+  - in src
+  - in test
+- Importing Gleam src code into test
+
+## Quick start
+
+```sh
+gleam run
+gleam test
+```
```

Comment:
This README file doesn't seem to be accurate

Reply:
Not 100% sure what we'd want to cover here, maybe something along the lines of:

> This project is to test various Deno runtime configuration settings.

Reply:
Readme has now been updated 🎉